soc/xtensa/intel_adsp/common/ipc.c

```diff
@@ -50,24 +55,31 @@ void z_intel_adsp_ipc_isr(const void *devarg)
 
 		regs->tdr = INTEL_ADSP_IPC_BUSY;
 		if (done && !IS_ENABLED(CONFIG_SOC_INTEL_CAVS_V15)) {
+#ifdef CONFIG_SOC_SERIES_INTEL_ACE1X
+			regs->tda = ACE_IPC_TDA1X_DONE;
```

Comment:
On ACE TDA is acknowledged by clearing bit 31 (BUSY). Here is writing value 1 to TDA which, of course, clears bit 31 and that is why it works but is misleading. It is better to just write 0.
Or, alternatively, something like ACK_TDA() macro could be defined which for ACE will write 0 into TDA with an explanation what that 0 is doing. As a bonus fewer "#ifdef CONFIG_SOC_SERIES_INTEL_ACE1X" will be needed in code.